### PR TITLE
[Issue 77] Minor styling fixes

### LINKED ISSUE #77: Minor styling fixes
- [x] Fix overloaded + icons (collapse and add)
- [x] Remove ripple
- [x] Multi-select menu item alignment
- [x] phantom widget under multiselect (see dark gray rect, below)
- [x] bump z-index for dropdown (see menu overlap with code editor, below)

style

### LINKED ISSUE #87: Safari doesn't support HTMLContentElement.getDistributedNodes()
`urth-viz-query` doesn't render in Safari due to lack of support for HTMLContentElement.getDistributedNodes(), see [these docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLContentElement/getDistributedNodes).

```javascript
this.$.myDataframe.getDistributedNodes is not a function
```

### PR DESCRIPTION
**Pull request by dalogsdon**
Resolves #77 
Resolves #87

### CODE REVIEW

File: urth-viz-query.html [hunk 1]
```diff
@@ -519,7 +493,8 @@
 
         ready: function() {
             // move filters template to user-supplied urth-core-dataframe in light DOM
-            var dataframe = this.$.myDataframe.getDistributedNodes()[0];
+            // var dataframe = this.$.myDataframe.getDistributedNodes()[0];
```
Comment (peller) [inline urth-viz-query.html]: ok to remove this line entirely

File: urth-viz-query.html [hunk 2]
```diff
@@ -519,7 +493,8 @@
...
+            var dataframe = Polymer.dom(this.$.myDataframe).getDistributedNodes()[0];
```
Comment (peller) [inline urth-viz-query.html]: commit comment should also ref #87

File: urth-viz-query.html (hunk 1 above)
Comment (dalogsdon) [inline urth-viz-query.html]: Didn't mean to check this in here, but I'll clean it up and we can make this fix #87  too

**Comment by dalogsdon** [comment]
Updated per @peller's comments

**Pull request merged**